src/frontend/src/lib/components/views/CreateAccount.svelte

```diff
@@ -1,24 +1,34 @@
 <script lang="ts">
   import { onMount } from "svelte";
   import FeaturedIcon from "$lib/components/ui/FeaturedIcon.svelte";
-  import { UserPlusIcon } from "@lucide/svelte";
+  import { PlusIcon } from "@lucide/svelte";
   import Button from "$lib/components/ui/Button.svelte";
   import Input from "$lib/components/ui/Input.svelte";
+  import { t } from "$lib/stores/locale.store";
+  import Checkbox from "$lib/components/ui/Checkbox.svelte";
   import ProgressRing from "$lib/components/ui/ProgressRing.svelte";
 
   interface Props {
-    create: (name: string) => void;
+    namesInUse: string[];
```

Comment:
It's a bit confusing, `namesInUse` and `isNameInUse`. How about: `existingNames`, `nameExists`?